docs/README.md

```diff
@@ -1,50 +1,91 @@
 # Rudr documentation
 
+*Rudr* is the Kubernetes reference implementation of the [Open Application Model](https://github.com/oam-dev/spec) (OAM) specification, a team-centric standard for building cloud-native apps, where:
+
+<table border=0 rules=none>
+<tr>
+<td><img alt="Developer role" src="./media/developer-role.png" /></td>
+<td><b>Developers</b> define application <b>components</b>,</td>
+</tr>
+<tr>
+<td><img alt="Application operator role" src="./media/app-operator-role.png" /></td>
+<td><b>Application operators</b> create instances of those components and assign them to <b>application configurations</a>, and</td>
+</tr>
+<tr>
+<td><img alt="Infrastructure operator role" src="./media/infra-operator-role.png" /></td>
+<td><b>Infrastructure operators</b> declare, install, and maintain the <b>underlying services</b> available on the platform.</td>
+</tr>
+</table>
+
+<img align="right" alt="Components and applications are deployed in the Rudr runtime within your Kubernetes cluster." src="./media/runtime.png" />
+
+To get started, check out the *Install Rudr* guide and the *Tutorial*. The *Concepts* guides will walk you through all the available options Rudr provides to run and manage applications, and the *How-Tos* cover more advanced scenarios.
+
 ## [Install Rudr](./setup/install.md)
 Install the Rudr runtime and its dependencies.
 
+## [Tutorial](./tutorials/deploy_and_update.md)
+Learn how to deploy, inspect, and update a Rudr application.
+
 ## Concepts
```

Comment:
Small nit - only thing that looks a little bit off is that "Install Rudr" and "Tutorial" are links with the same heading size as Concepts, which is a separate section. 

Maybe we could put the "Install Rudr" and "Tutorial" links under a section called "Getting Started" so it looks more like the Concepts section?